src/recommendations/RecommendationsPage.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+
+import { getConfig } from '@edx/frontend-platform';
+import { injectIntl } from '@edx/frontend-platform/i18n';
+import {
+  Hyperlink, Image, StatefulButton,
+} from '@edx/paragon';
+import PropTypes from 'prop-types';
+
+import { DEFAULT_REDIRECT_URL } from '../data/constants';
+import messages from './messages';
+import RecommendationsList from './RecommendationsList';
+
+const recommendationData = [
+  {
+    key: 'course-key1',
+    title: 'Introduction to Blockchain',
+    bannerImageUrl: 'https://prod-discovery.edx-cdn.org/media/course/image/894bd8b1-7f01-4004-946e-b29f3a3bb282-b656f554c388.small.png',
+    owners: [
+      {
+        key: 'test-owner-key1',
+        name: 'IBM',
+        logoImageUrl: 'https://prod-discovery.edx-cdn.org/organization/logos/87b07564-d569-4cfd-bee6-8b0a407acb73-dc33e4b5f353.png',
+      },
+      {
+        key: 'test-owner-key12',
+        name: 'IBM1',
+        logoImageUrl: 'https://prod-discovery.edx-cdn.org/organization/logos/87b07564-d569-4cfd-bee6-8b0a407acb73-dc33e4b5f353.png',
+      },
+    ],
+    marketingUrl: 'https://www.edx.org/course/introduction-to-blockchain',
```

Comment:
This is not the response we are getting from Algolia. Can you work with @mubbsharanwar to see what the index returns?

Reply:
@zainab-amir I have co-ordinated with @mubbsharanwar and updated the response according to the Algolia.